app/Http/Kernel.php

```diff
@@ -45,7 +45,6 @@ class Kernel extends HttpKernel
         ],
 
         'api' => [
-            'throttle:120,1',
```

Comment:
I think I get it; at the Kernel level we're not going to be able to reference the `.env`. So we don't try.

Instead, we explictly load in the throttle middleware later on, still in the early boot sequence so it's properly baked-in, but not so soon that we can't read the `.env`. Love it.

I was completely wrong about this. This is extremely clean. Beautifully done!